README.md

```diff
@@ -69,86 +69,73 @@ For a deeper dive into HSSM, please follow [our main tutorial](https://lnccbrown
-
-To create a conda environment, use the following command. Substitute `mamba` for `conda` if `mamba` is available:
+Use the following command to install HSSM into your virtual environment:
 
 ```bash
-conda create -n <your-env-name> python=3.11
-conda activate <your-env-name>
+conda install -c conda-forge hssm
 ```
 
-Substitute `<your-env-name>` with the name of the virtual environment that you choose. HSSM 0.2.0 and above supports Python versions 3.10 and 3.11.
-
-### Step 2: Install PyMC through conda-forge
+### Install HSSM on Linux and MacOS (with GPU Support)
 
-Installation through `conda-forge` is the official way of installing PyMC. This will also install other libraries such as `libblas` that PyMC requires to run properly.
+If you need to sample with GPU, please install JAX with GPU support before installing HSSM:
 
 ```bash
-conda install -c conda-forge pymc
+conda install jaxlib=*=*cuda* jax cuda-nvcc -c conda-forge -c nvidia
+conda install -c conda-forge hssm
 ```
 
-### [Optional] Install JAX with CUDA support
+### Install HSSM on Windows (CPU only)
 
-If you need to sample with GPU, please install JAX with GPU support after installing PyMC following one of the two commands below:
-
-#### Option 1: Install JAX with CUDA support via `conda`
-
-At the moment, there is a community supported conda build on conda-forge:
+Because `jaxlib` is not available through `conda-forge` on Windows, you need to install JAX on Windows through `pip` before getting HSSM:
 
 ```bash
-conda install jaxlib=*=*cuda* jax cuda-nvcc -c conda-forge -c nvidia
+pip install jax
+conda install -c conda-forge hssm
 ```
 
-#### Option 2: Install JAX with CUDA support via `pip`
+### Install HSSM on Windows (with GPU support)
 
-Installing `jax` via `pip` should also work:
+You simply need to install JAX with GPU support before getting HSSM:
 
 ```bash
 pip install jax[cuda12]
+conda install -c conda-forge hssm
 ```
 
-#### Support for Apple Silicon, AMD, and other GPUs
+### Support for Apple Silicon, AMD, and other GPUs
 
-JAX also has support other GPUs. Please follow the [Official JAX installation guide](https://jax.readthedocs.io/en/latest/installation.html) to install the correct version of JAX before proceeding.
+JAX also has support other GPUs. Please follow the [Official JAX installation guide](https://jax.readthedocs.io/en/latest/installation.html) to install the correct version of JAX before installing HSSM.
 
-### Step 3: Install `hssm` through `pip`
-
-In the same environment, install `hssm` through `pip`.
-
-```bash
-pip install hssm
-```
 
 ## Advanced Installation
 
 ### Install HSSM directly with Pip
```

Comment:
maybe mention here that even if you want to install `hssm` via pip, one should still install `pymc` first via conda, otherwise issue can arise.

Reply:
I think the `note` section below addresses this?